src/components/Formik/FormikRadioGroup/FormikRadioGroup.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {
+  FC,
+  ChangeEvent,
+  FocusEvent,
+} from 'react';
+import RadioGroup from '../../RadioGroup/RadioGroup';
+import { Option } from '../../RadioGroup/RadioGroupTypes';
```

Comment:
These types ended getting shared between RadioGroup and the Formik component. Should there be a dir in the root to store these files? Or, should RadioGroup retain ownership of them?

Is the naming convention good?

Reply:
I think it's reasonable for RadioGroup to retain ownership.

Reply:
i think because FormikRadioGroup is using RadioGroup, its ok for RadioGroup to own those types